### DISCUSSION TITLE
Component Library / Cataloging

### DISCUSSION THREAD

**Discussion by hariDasu** [NJIT-IS332-Fall2023/is322-next-app#18] [Ideas]
https://blog.logrocket.com/ladle-storybook-performance-project-sizes/

- a tool like ladle/storybook can be time consuming to setup/integrate but can help to develop components in isolation and allows new developers to see and interact with components without having to navigate to their location in the applicaiton

- storybook seems the oldest but has some flaws, some work could be done to evaluate what the best tool for our codebase is currently
  -  potentialy opening RFC for discussion, or class discussion/standup meeting poll for sake of brevity

- [x] https://github.com/NJIT-IS332-Fall2023/is322-next-app/issues/10
- [ ] https://github.com/NJIT-IS332-Fall2023/is322-next-app/issues/13

**Comment by benjamin-shuster** [comment]
https://storybook.js.org/recipes/next/
https://ladle.dev/docs/nextjs/

these both mention the setup / caveats of using these tools with Next.js  might be worth taking into consideration

**Comment by hariDasu** [comment]
> https://storybook.js.org/recipes/next/
> 
> https://ladle.dev/docs/nextjs/
> 
> 
> 
> these both mention the setup / caveats of using these tools with Next.js  might be worth taking into consideration 

Ouch! If it will take too much work to setup might be worth delaying until after 2.0, total # of components in mvp should be minimized anyway
